cache/src/main/java/org/apollo/cache/decoder/NpcDefinitionDecoder.java

```diff
@@ -1,14 +1,11 @@
 package org.apollo.cache.decoder;
 
-import java.io.IOException;
-import java.io.UncheckedIOException;
-import java.nio.ByteBuffer;
-import java.util.Arrays;
-
-import org.apollo.cache.IndexedFileSystem;
-import org.apollo.cache.archive.Archive;
+import it.unimi.dsi.fastutil.ints.Int2ObjectArrayMap;
```

Comment:
Use standard collections instead of these `fastutil` collections. We optimize on a case by case basis, not generally like this.

Reply:
That's fine. I'll get rid of it.